config/sync/user.role.su_site_embedder.yml

```diff
@@ -0,0 +1,14 @@
+uuid: 9dedbe8e-27b6-40de-95fd-29e4efe67be4
+langcode: en
+status: true
+dependencies:
+  module:
+    - field_permissions
+id: su_site_embedder
```

Comment:
can you add a couple codeception tests for this role

Reply:
can do.